app/javascript/src/Navigation/components/ContextSelector.jsx

```diff
@@ -5,28 +5,95 @@ import 'core-js/modules/es6.map'
 import 'core-js/es6/array'
 import React from 'react'
 import { render } from 'react-dom'
-import { ApiSearch } from './ApiSearch'
 import getActiveMenuTitle from '../utils/getActiveMenuTitle'
 
+import '../styles/ContextSelector.scss'
+
 const DASHBOARD_PATH = '/p/admin/dashboard'
 const AUDIENCE_PATH = '/buyers/accounts'
 
-const ContextSelector = ({ apis, currentApi, activeMenu }) => (
-  <div className="PopNavigation PopNavigation--context">
-    <a className="PopNavigation-trigger u-toggler" href="#context-menu" title="Context Selector">
-      <span> {getActiveMenuTitle(activeMenu, currentApi)} <i className='fa fa-chevron-down' /></span>
-    </a>
-    <ul id="context-menu" className="PopNavigation-list u-toggleable">
-      <li className="PopNavigation-listItem">
-        <a className="PopNavigation-link" href={DASHBOARD_PATH}>Dashboard</a>
+const apiPathRoot = apiId => `/apiconfig/services/${apiId}`
+
+class ContextSelector extends React.Component {
+
+  constructor (props) {
+    super(props)
+    this.state = {
+      filterQuery: ''
+    }
+  }
+
+  onFilterChange (event) {
+    const filterQuery = event.target.value.toLowerCase()
+    this.setState({ filterQuery })
+  }
+
+  renderInput () {
+    const { apis = [] } = this.props
+
+    if (apis.length < 2) {
+      return null
+    }
+
+    return (
+      <li className="PopNavigation-listItem nav-search-widget docs-search" id="context-widget">
+        <input
+          onChange={e => this.onFilterChange(e)}
+          type="search"
+          className="docs-search-input"
```

Comment:
Doesn't belong to this PR really, but just wondering if it would make sense to rename this class to something more generic, as it's not only used for Docs search now. Maybe `search-input`? :) cc @thomasmaas

Reply:
yes but no time. next iteration.